{"name": "Создание двух одинаковых курьеров", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидалось сообщение об ошибки 'Этот логин уже используется'\nassert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  \n  - Этот логин уже используется\n  + Этот логин уже используется. Попробуйте другой.", "trace": "self = <tests.test_creating_courier.TestCreatingCourier object at 0x0000022E9C39F1D0>, api_client = <api_client.api_client.APIClient object at 0x0000022E9C6A27B0>\ndelete_courier = {'firstName': 'svbrrnfmcu', 'login': 'yzmjjdsbnj', 'password': 'oprfxnqbet'}\n\n    @allure.title(\"Создание двух одинаковых курьеров\")\n    @allure.description(\n        \"Проверка, что создать двух одинаковых курьеров невозможно, возвращение кода 409 и сообщения об ошибки в теле ответа\")\n    def test_double_creating_courier_conflict(self, api_client, delete_courier):\n        payload = delete_courier\n        with allure.step(\"Отправка POST-запроса для создания курьера\"):\n            response = api_client.post(St.ENDPOINT_COURIER, data=payload)\n    \n        with allure.step(\"Проверка, что код ответа на создание заказа 201\"):\n            assert response.status_code == 201 , \"Ожидался успешный запрос\"\n    \n        with allure.step(\"Отправка POST-запроса для создания курьера с теме же данными\"):\n            double_response = api_client.post(St.ENDPOINT_COURIER, data=payload)\n    \n        with allure.step(\"Проверка, что код ответа на создание заказа 409\"):\n            assert double_response.status_code == St.RESPONSE_COURIER_CONFLICT[\"code\"],\\\n                f\"Ожидалась ошибка {St.RESPONSE_COURIER_CONFLICT[\"code\"]} при создание курьера с такими же данными\"\n    \n        with allure.step(\"Проверка, что в теле ответа сообщение об ошибки\"):\n>           assert double_response.json()[\"message\"] == St.RESPONSE_COURIER_CONFLICT[\"message\"],\\\n                f\"Ожидалось сообщение об ошибки '{St.RESPONSE_COURIER_CONFLICT[\"message\"]}'\"\nE           AssertionError: Ожидалось сообщение об ошибки 'Этот логин уже используется'\nE           assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\nE             \nE             - Этот логин уже используется\nE             + Этот логин уже используется. Попробуйте другой.\n\ntests\\test_creating_courier.py:44: AssertionError"}, "description": "Проверка, что создать двух одинаковых курьеров невозможно, возвращение кода 409 и сообщения об ошибки в теле ответа", "steps": [{"name": "Отправка POST-запроса для создания курьера", "status": "passed", "start": 1739101704358, "stop": 1739101704480}, {"name": "Проверка, что код ответа на создание заказа 201", "status": "passed", "start": 1739101704480, "stop": 1739101704480}, {"name": "Отправка POST-запроса для создания курьера с теме же данными", "status": "passed", "start": 1739101704480, "stop": 1739101704597}, {"name": "Проверка, что код ответа на создание заказа 409", "status": "passed", "start": 1739101704597, "stop": 1739101704597}, {"name": "Проверка, что в теле ответа сообщение об ошибки", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидалось сообщение об ошибки 'Этот логин уже используется'\nassert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  \n  - Этот логин уже используется\n  + Этот логин уже используется. Попробуйте другой.\n", "trace": "  File \"C:\\Users\\user\\Pycharm\\Sprint_7\\tests\\test_creating_courier.py\", line 44, in test_double_creating_courier_conflict\n    assert double_response.json()[\"message\"] == St.RESPONSE_COURIER_CONFLICT[\"message\"],\\\n"}, "start": 1739101704597, "stop": 1739101704597}], "start": 1739101704358, "stop": 1739101704598, "uuid": "5b625d71-d9ce-42f6-beb9-1f942d116b13", "historyId": "cd457c3a42c0ab4f3957a5263f7a12e3", "testCaseId": "cd457c3a42c0ab4f3957a5263f7a12e3", "fullName": "tests.test_creating_courier.TestCreatingCourier#test_double_creating_courier_conflict", "labels": [{"name": "story", "value": "Создание курьера"}, {"name": "feature", "value": "Работа с курьером"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creating_courier"}, {"name": "subSuite", "value": "TestCreatingCourier"}, {"name": "host", "value": "bnovo-nb-0056"}, {"name": "thread", "value": "19372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creating_courier"}]}